src/xcode/ENA/ENA/Source/Services/Risk/Provider/RiskProvider.swift

```diff
@@ -473,7 +479,10 @@ extension RiskProvider {
 				numberOfDaysWithLowRisk: risk.details.numberOfDaysWithRiskLevel,
 				numberOfDaysWithHighRisk: 0,
 				calculationDate: Date(),
-				riskLevelPerDate: [Date(): .low]
+				riskLevelPerDate: [
+					today: .low,
+					someDaysAgo: .high
```

Comment:
If this mock should represent a low risk level, then all riskLevelsPerDate have to be .low
```suggestion
					someDaysAgo: .low
```